Adyen/Core/Components/AbstractPersonalInformationComponent/PhoneFormItemInjector.swift

```diff
@@ -12,29 +12,38 @@ internal final class PhoneFormItemInjector: FormItemInjector, Localizable {
 
     internal let style: FormTextItemStyle
 
-    internal let phoneExtensions: [PhoneExtensionPickerItem]
+    internal let phoneExtensions: [PhoneExtension]
 
     internal var value: PhoneNumber?
 
     internal var identifier: String
+    
+    internal let presenter: WeakReferenceViewControllerPresenter
```

Comment:
What is this protocol/class? Can't find it on current code

Reply:
It's there for longer already - it's a wrapper that holds a weak reference to a presenter and is itself conforming to the protocol - this way we don't have to make the parameter nil-able